spec/services/zendesk_intake_service_spec.rb

```diff
@@ -307,6 +309,73 @@
   end
 
   describe "#new_ticket_group_id" do
+    context "when there is a source parameter" do
+      context "when there is also a state" do
+        let(:source) { "uwkc" }
+        let(:state) { "ak" }
```

Comment:
two other specs that would help:
- when there is a souce but it does not match any organization (for example: "propel")
- when there is a source that is longer than just the organization code (but contains the org code in the beginning, for example: "uwkc_flyer")